libraries/AP_LeakDetector/AP_LeakDetector.cpp

```diff
@@ -19,6 +19,14 @@ const AP_Param::GroupInfo AP_LeakDetector::var_info[] = {
     // @User: Standard
     AP_GROUPINFO("1_LOGIC", 2, AP_LeakDetector, _default_reading[0], 0),
 
+    // @Param: 1_TYPE
+    // @DisplayName: Pin type (analog/digital)
+    // @Description: NOT FOR USE with Pixhawk, Pixhawk 4 or Navigator flight controllers. Enables manual leak-pin selection on other boards-- use this parameter to indicate the signal type (0:analog, 1:digital) of an appropriately configured pin, then use the LEAK1_PIN parameter to specify its number.
```

Comment:
Let's move this, "NOT FOR USE" portion to closer to the end of the parameter description.